tests/src/test/java/alluxio/server/ft/journal/raft/EmbeddedJournalIntegrationTestFaultTolerance.java

```diff
@@ -77,6 +87,52 @@ public void failover() throws Exception {
     mCluster.notifySuccess();
   }
 
+  @Test
+  public void syncMetadataFailOver() throws Exception {
```

Comment:
nit: include embedded journal in name
```suggestion
  public void syncMetadataEJFailOver() throws Exception {
```